torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -705,6 +718,22 @@ struct GraphFuser {
     for (size_t i = 0; i < nchunks; i++) {
       bchunk->eraseOutput(nchunks * producer_index);
     }
+
+    // The output of producer_for_chunk_node could have been used in some aten::size
+    // operators, so we need to clean those up as well (we simply broadcast all its tensor inputs).
+    auto size_calc_uses = producer_for_chunk_node->output()->uses();
+    if (!size_calc_uses.empty()) {
+      auto tensor_inputs = filter(producer_for_chunk_node->inputs(),
+                                  [](Value * v) { return v->type()->isSubtypeOf(DynamicType::get()); });
+      auto tensor_sizes = fmap(tensor_inputs,
```

Comment:
nit: SymbolicVariable::size could be good to have as well

Reply:
Sure. I can add it if I'll have to add any changes to this PR, but I'd really like to merge it today so that it makes it into the release.